kivy/core/window/__init__.py

```diff
@@ -933,6 +933,34 @@ def _set_top(self, value):
     the position set in :class:`~kivy.config.Config`.
     '''
 
+    def _get_opacity(self):
+        return self._get_window_opacity()
+
+    def _set_opacity(self, opacity):
+        self._set_window_opacity(opacity)
+        return True
+
+    def _get_window_opacity(self):
+        Logger.warning('Window: Opacity is not implemented in the current '
+                       'window provider')
+
+    def _set_window_opacity(self, opacity):
+        Logger.warning('Window: Opacity is not implemented in the current '
+                       'window provider')
+
+    opacity = AliasProperty(_get_opacity, _set_opacity)
```

Comment:
Change to `opacity = AliasProperty(_get_opacity, _set_opacity, cache=True)` so that opacity value is cached so that cached value inside of `AliasProperty` will be returned instead of calling getter method.

Reply:
Done.